types/heremaps/index.d.ts

```diff
@@ -4605,7 +4606,140 @@ declare namespace H {
          * This type encapsulates a response object provider by a HERE platform service.
          */
         interface ServiceResult {
-            [key: string]: string;
+            [key: string]: any;
```

Comment:
Do we need to keep this index? Originally, I put this in as this was all the detail provided by the HERE docs, and I hadn't thought to look at the structure of the result myself! :P